unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/DecentralandEntity.cs

```diff
@@ -22,6 +22,7 @@ public class DecentralandEntity : IDCLEntity
         public Dictionary<System.Type, ISharedComponent> sharedComponents { get; private set; } = new Dictionary<System.Type, ISharedComponent>();
 
         public GameObject gameObject { get; set; }
+        public Transform transform { get; set; }
```

Comment:
`transform` is already in `gameObject`. Why do we need this?

Reply:
I thought it will be more "Unity-friendly". Unity monobehaviours have some properties added like Transform, GameObject...etc. I think that for a new unity developer it will be more intuitive to find the transform in the entity rather than go to the gameObject to find the transform. Let me know what do you think